Makefile

```diff
@@ -27,7 +27,7 @@ all: sed-tools
 	$(GCC) $(CFLAGS) -o $@ $<
 
 sed-tools: $(OBJECTS)
-	$(GCC) $(LDFLAGS) -o $@ $(OBJECTS)
+	$(GCC) -o $@ $(OBJECTS) $(LDFLAGS)
```

Comment:
There was an issue with this?

Reply:
Yeah the order of the gcc options seemed to matter. Like if we put the linker options where they were originally, we would get linker issues (the libcrypto and libssl bits). But when u put the link options last it would compile successfully. I think typically you put the linker options after you list the objects that needs to be linked.

Reply:
Looks like this doesn't affect new versions of gcc. Good find